src/utils/theme.tsx

```diff
@@ -35,6 +35,11 @@ export const theme = createTheme({
     10: 40,
     12: 48,
     14: 60,
+    15: 80,
+    16: 100,
+    17: 120,
+    18: 140,
+    19: 160,
```

Comment:
These numbers are meant to be multiples of 4. It would be good if we kept them that way.